superset/constants.py

```diff
@@ -133,14 +133,14 @@ class RouteMethod:  # pylint: disable=too-few-public-methods
     "time_column": "time_column",
     "time_grain": "time_grain",
     "time_range": "time_range",
+    "druid_time_origin": "druid_time_origin",
+    "time_grain_sqla": "time_grain_sqla",
+    "time_range_endpoints": "time_range_endpoints",
```

Comment:
For legacy plugins (`viz.py`), these have to be mapped into the main `form_data`, as they are later mapped to `extras` where they should ultimately go:
https://github.com/apache/superset/blob/6119d8e998601d214030e61eeed8b1709342fa3b/superset/viz.py#L363-L370